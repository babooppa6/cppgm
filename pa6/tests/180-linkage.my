recog 1
tests/180-linkage.t BAD
