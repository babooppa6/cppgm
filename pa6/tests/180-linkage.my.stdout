------ replace Text-extern--"C"--void--foo-(-)-;-extern--"D"-{--void--bar-(-)-;--void--baz-(-)-;-}-;-
